WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductCoordinator.swift

```diff
@@ -408,7 +408,6 @@ private extension AddProductCoordinator {
         BottomSheetPresenter(configure: { bottomSheet in
             var sheet = bottomSheet
             sheet.prefersEdgeAttachedInCompactHeight = true
-            sheet.largestUndimmedDetentIdentifier = .none
```

Comment:
From what I understand, we want the content below the sheet to not be interactable (the dimmed background hints that). According to the documentation for (`largestUndimmedDetentIdentifier`)[https://developer.apple.com/documentation/uikit/uisheetpresentationcontroller/3858107-largestundimmeddetentidentifier], when we set this to `.none` it means the dimmed background is never displayed. Removing this resolves the issue - we can also explicitly set it to `nil` to clarify the intention, but it's redundant since the default value is `nil`.